specs/linux/apparmor_profiles.table

```diff
@@ -0,0 +1,13 @@
+table_name("apparmor_profiles")
+description("Track active AppArmor profiles.")
+schema([
+    Column("path", TEXT, "Path", index=True),
```

Comment:
Ahh, so sorry @alessandrogario, I did not notice this before but we should add more context to the table descriptions.